build/ci/evergreen.yml

```diff
@@ -16,6 +16,9 @@ exec_timeout_secs: 4200 # automatically fail any task if it takes longer than 1h
 stepback: true
 command_type: system
 pre_error_fails_task: true
+include:
+  - filename: build/ci/release.yml
+  - filename: build/ci/evergreen_common.yml
 
 variables:
   - &go_linux_version
```

Comment:
some are already defined in common ( e.g. &go_env), can we remove them from here?

Reply:
as mentioned earlier we have to have them on both files

Reply:
same for release.yml